.github/workflows/CI.yml

```diff
@@ -76,12 +77,21 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
         with:
-          python-version: "3.x"
+          python-version: |
+            3.8
+            3.9
+            3.10
+            3.11
+            3.12
+            3.13
+            pypy3.9
+            pypy3.10
+          allow-prereleases: true
       - name: Build wheels
         uses: PyO3/maturin-action@v1
         with:
           target: ${{ matrix.target }}
-          args: --release --out dist --interpreter '3.8 3.9 3.10 3.11 3.12 pypy3.8 pypy3.9 pypy3.10'
```

Comment:
Can we keep pypy3.8 so long as we support CPython 3.8?

Reply:
Oh, you mentioned that in your description, thanks, that makes sense, so never mind this.